doc/build_wasm_app.md

```diff
@@ -3,7 +3,7 @@
 Prepare WASM building environments
 ==================================
 
-For C and C++, WASI-SDK version 12.0+ is the major tool supported by WAMR to build WASM applications. Also, we can use [Emscripten SDK (EMSDK)](https://github.com/emscripten-core/emsdk), but it is not recommended. And there are some other compilers such as the standard clang compiler, which might also work [here](./other_wasm_compilers.md).
+For C and C++, WASI-SDK version 12.0+ is the major tool supported by WAMR to build WASM applications. Also, we can use [Emscripten SDK (EMSDK)](https://github.com/emscripten-core/emsdk), but it is not recommended. And there are some other compilers such as the standard clang compiler, which might also work [here](other_wasm_compilers.md).
```

Comment:
Must we change the path of other_wasm_compilers.md?

Reply:
yes, somehow, gitbook can't recognize the former one

Reply:
But it supports `../../doc/wamr_api.md` in gitbook/advance_tutorial/README.md :)

Reply:
I tried to change it back to `./other_wasm_compilers.md`, it works fine now. It's rather strange that some other link has displayed similar behavior. It's not working where it's supposed to work, but when I modify it (any modification) and change it back, it will work fine. Now I am going through the whole document to check whether other links are fine.